compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -559,20 +559,24 @@ public BLangNode transform(ModulePartNode modulePart) {
 
     @Override
     public BLangNode transform(ModuleVariableDeclarationNode modVarDeclrNode) {
+
         TypedBindingPatternNode typedBindingPattern = modVarDeclrNode.typedBindingPattern();
+
         BindingPatternNode bindingPatternNode = typedBindingPattern.bindingPattern();
+        Optional<io.ballerina.compiler.syntax.tree.ExpressionNode> initializer = modVarDeclrNode.initializer();
```

Comment:
Shall we directly call `modVarDeclrNode.initializer()` in L592 instead? We don't seem to be using this variable elsewhere.

Reply:
+1